x-pack/plugins/security_solution/public/common/components/top_n/top_n.tsx

```diff
@@ -86,7 +87,15 @@ const TopNComponent: React.FC<Props> = ({
   );
   const indicesSelector = useMemo(getIndicesSelector, []);
   const { all: allIndices, raw: rawIndices } = useSelector<State, IndicesSelector>(
-    (state) => indicesSelector(state),
+    (state) =>
+      indicesSelector(
+        state,
+        timelineId != null
```

Comment:
@XavierM why are we even offering a raw events option in the alert view tho? the sourcerer wont have any raw indices

Reply:
in case user want to switch the view of the top-n from alerts to raw events

Reply:
by the way I will check the app to see if this logic is working as expected, maybe it will be nice to have a function and unit test to see if this logic is working as expected